operator/cmd/mesh/install.go

```diff
@@ -160,35 +164,47 @@ func runApplyCmd(cmd *cobra.Command, rootArgs *rootArgs, iArgs *installArgs, log
 	if err := configLogs(logOpts); err != nil {
 		return fmt.Errorf("could not configure logs: %s", err)
 	}
-	if err := InstallManifests(setFlags, iArgs.inFilenames, iArgs.force, rootArgs.dryRun,
-		iArgs.kubeConfigPath, iArgs.context, iArgs.readinessTimeout, l); err != nil {
+	iop, err := InstallManifests(applyFlagAliases(iArgs.set, iArgs.manifestsPath, iArgs.revision), iArgs.inFilenames, iArgs.force, rootArgs.dryRun,
+		iArgs.kubeConfigPath, iArgs.context, iArgs.readinessTimeout, l)
+	if err != nil {
 		return fmt.Errorf("failed to install manifests: %v", err)
 	}
 
+	if iArgs.verify {
+		l.LogAndPrint("\nVerifying installation")
+		installationVerifier := verifier.NewStatusVerifier(iop.Namespace, iArgs.manifestsPath, iArgs.kubeConfigPath,
+			iArgs.context, iArgs.inFilenames, clioptions.ControlPlaneOptions{Revision: iop.Spec.Revision}, l)
+		if err := installationVerifier.Verify(); err != nil {
+			return fmt.Errorf("verification failed with the following error: %v\n\n"+
+				"Please run `istioctl verify-install` manually", err)
```

Comment:
i'm not sure that running verify-install manually would help here - i suggest dropping that sentence.

Reply:
Dropping it. I meant, if it fails due to some reason - probably we can have a situation where deployment is not ready in time. In that case, users can run `verify-install` later.